src/components/Media/index.js

```diff
@@ -92,48 +98,47 @@ export default class Media extends PureComponent {
     ])
   }
 
-  renderImage(props) {
-    return props.icon === loaded ? (
+  renderImage() {
+    return this.isLoaded ? (
       <img
-        {...compose(props.theme.img)}
-        src={props.src}
-        alt={props.alt}
-        width={props.width}
-        height={props.height}
+        {...compose(this.props.theme.img)}
+        src={this.props.src}
+        alt={this.props.alt}
+        width={this.props.width}
+        height={this.props.height}
       />
     ) : (
       <svg
-        {...compose(props.theme.img)}
-        width={props.width}
-        height={props.height}
+        {...compose(this.props.theme.img)}
+        width={this.props.width}
+        height={this.props.height}
         ref={ref => (this.dimensionElement = ref)}
       />
     )
   }
 
-  renderNoscript(props) {
-    return props.ssr ? (
+  renderNoscript() {
+    return this.props.ssr ? (
```

Comment:
this change is unrelated to the feature. Idea is to move to hooks from classes, so this code will be changed back eventually.

Reply:
The only reason was to use the `isLoaded` function/getter properly in all other cases, to be consistent.